include/DistributedDomain.hpp

```diff
@@ -424,6 +424,11 @@ class DistributedDomain : public Domain<T, Dataset, Tree>
         // MUST DO SYNCHALOS AND BUILDTREE AFTER
 
         d.count = workAssigned;
+
+#if defined(USE_CUDA)
```

Comment:
why not ifdef ?

Reply:
Just in case we needed an other condition. With ifdef you can only check for one flag. If we need to check for USE_CUDA and USE_MPI it'll be cumbersome using ifdefs.

Reply:
ok, then I would update the other ifdef to have the same everywhere.